README.md

```diff
@@ -0,0 +1,90 @@
+# Linkerd RFCs
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow.
+
+Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the Linkerd community.
+
+The "RFC" (request for comments) process is intended to provide a consistent and controlled path for contributions to enter the project, so that all stakeholders can be confident about the project direction and maintainability of the codebase.
+
+## When you need to follow this process
+
+[when you need to follow this process]: #when-you-need-to-follow-this-process
+
+You need to follow this process if you intend to make "substantial" changes to Linkerd2 (Linkerd2-proxy, Linkerd Control Plane) or the RFC process itself. What constitutes an "substantial" is a foundational concern for the Linkerd project.
+
+Similarly, any technical effort (refactoring, major architectural change) that will impact a large section of the development community should also be communicated widely. The Linkerd RFC process is suited for this even if it will have zero impact on the typical user or operator.
+
+## Before creating an RFC
+
+[before creating an rfc]: #before-creating-an-rfc
+
+A hastily-proposed RFC can hurt its chances of acceptance. Low quality proposals, proposals for previously-rejected features, or those that don't fit into the near-term roadmap, may be quickly rejected, which can be demotivating for the unprepared contributor. Laying some groundwork ahead of the RFC can make the process smoother.
+
+Although there is no single way to prepare for submitting an RFC, it is generally a good idea to pursue feedback from other project developers beforehand, to ascertain that the RFC may be desirable; having a consistent impact on the project requires concerted effort toward consensus-building.
+
+The most common preparations for writing and submitting an RFC include talking the idea over on our [Linkerd Slack, #contributors](https://slack.linkerd.io), or discussing the topic on our [CNCF developer mailing list](https://lists.cncf.io/g/cncf-linkerd-dev). You may file issues on this repo for discussion, but these are not actively looked at by the teams.
+
+As a rule of thumb, receiving encouraging feedback from long-standing project developers is a good indication that the RFC is worth pursuing.
+
+## What the process is
+
+[what the process is]: #what-the-process-is
+
+In short, to get a major feature added to Linkerd, one must first get the RFC merged into the RFC repository as a markdown file. At that point the RFC is "active" and may be implemented with the goal of eventual inclusion into Linkerd.
+
+* Fork the [RFC repository](https://github.com/linkerd/rfc) * Copy `rfc-template.md` to `active/my-contribution.md` (where "my-contribution" is
+
+ descriptive.).
```

Comment:
How would you feel about requiring a numbering scheme like `0000-linkerd-rfc-process.md`? Or perhaps `0000/rfc.md`? Basically, What if it makes sense to have auxiliary files? How do they get grouped together? Additionally, it would be nice to have some natural sorting and uniqueness.

Reply:
Added the RFC numbering schema.